pkg/reconciler/openshift/tektonchain/extension.go

```diff
@@ -42,7 +46,10 @@ func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Tr
 	return []mf.Transformer{
 		occommon.RemoveRunAsUser(),
 		occommon.RemoveRunAsGroup(),
+		occommon.RemoveRunAsUserForStatefulSet(tektonChainsControllerName),
+		occommon.RemoveRunAsGroupForStatefulSet(tektonChainsControllerName),
 		occommon.ApplyCABundles,
+		occommon.ApplyCABundlesForStatefulSet(tektonChainsControllerName),
```

Comment:
Don't we require this for all other components ex: Results (Its also statefulset isn't it ?

Reply:
Added for result watcher
We should add for al statefulsets

Reply:
> Added for result watcher

:+1: 

> We should add for al statefulsets

I see this PR address for Pipelines, Chains and Results with that we can say all components are covered ??

Reply:
yes all component with statefulset feature are covered